source/_docs/installation.markdown

```diff
@@ -27,7 +27,13 @@ Home Assistant provides multiple ways to be installed. A requirement is that you
     <div class='img-container'>
       <img src='/images/supported_brands/home-assistant.png' />
     </div>
-    <div class='title'>Hass.io<br>(Beginner friendly)</div>
+    <div class='title'>Hass.io on HassOS<br>(Beginner friendly)</div>
```

Comment:
No need to add HassOS as beginner shouldn't need to care about the underlying operating system.